ssg/oval.py

```diff
@@ -14,7 +14,7 @@
 from .xml import oval_generated_header
 from .id_translate import IDTranslator
 
-SHARED_OVAL = re.sub(r'shared.*', 'shared', __file__) + '/checks/oval/'
+SHARED_OVAL = re.sub(r'ssg/.*', 'shared', __file__) + '/checks/oval/'
```

Comment:
I'm intrigued by why `SHARED_OVAL` is defined this way, but I see that totally changing it is not part of this PR.